spring-cloud-gcp-pubsub/pom.xml

```diff
@@ -42,5 +42,34 @@
 			<artifactId>spring-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- SI channel adapters -->
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-core</artifactId>
```

Comment:
How about if we create another `spring-cloud-gcp-integration` module, alongside the existing `spring-cloud-gcp-core`, `spring-cloud-gcp-dependencies`, etc.?
I kinda like it, although there will be a little redundancy. Users (like me) might ask why the PubSub SI channel adapters aren't in the `spring-cloud-gcp-pubsub` module. Logically, it could make more sense to have `spring-cloud-gcp-pubsub/integration`, `spring-cloud-gcp-pubsub/cloud`, etc. They could still be arranged as different modules and it seems like the most coherent arrangement to me (unless there is a case where users could use importing all SI adapters at a time to their projects, in which case having a separate `integration` module could make more sense?).

Would be great to agree on a decision here asap!